wordpress-mobile/WordPress-iOS
# Add WordPress.com logo to that login screen

This should help confused users that end up on the WordPress.com login screen when they intended for self-hosted.

To test:
1. Launch the app without having any saved credentials
2. Enter a real WordPress.com username into the text field. Hit Next.
3. Ensure that the "WordPress.com" logotype appears.

Design proof: @mattmiklic 

![wpcom-logo-login](https://cloud.githubusercontent.com/assets/517257/19866882/90a76910-9f67-11e6-9977-8d4d1e28f441.png)

Needs review: @frosty
